modules/persistent-storage-csi-olm-operator-install.adoc

```diff
@@ -24,24 +41,42 @@ To install the {FeatureName} CSI Driver Operator from the web console:
 .. Locate the {FeatureName} CSI Operator by typing *{FeatureName} CSI* in the filter box.
```

Comment:
We probably need to rename `{FeatureName}` from "SMB CIFS CSI" to "CIFS/SMB CSI". I.e. if you try to type "SMB CIFS CSI" in the console, it finds nothing.

Reply:
yes agree let's be accurate with the name